html/js/product-multilingual.js

```diff
@@ -18,6 +18,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+/*eslint dot-location: "off"*/
```

Comment:
IMHO disabling a purely stylisting linter rule for a single file defeats the purpose of using style rules in general. We can definitely discuss the [`dot-location`](https://eslint.org/docs/rules/dot-location) rule for all files (and change it in `.eslintrc.js`), as it simply extends from the `eslint:recommended` rule set. I wouldn't want to disable it for one file, though, because then the style wouldn't be consitent among all JS files any more.

Reply:
i mean, it's not like anything else in product-multilingual.js is consistent, even within the file...

Reply:
but with dot-location in particular, a convention of having the dot on the end of the previous line is one i've not seen before, and doesn't make sense to me.

Reply:
As I said, it's in `eslint:recommended`. Changing it for just one file is super unexpected and shouldn't be done IMHO.

Reply:
well as i mentioned in my commit, it's going directly against the examples in the jquery docs https://api.jquery.com/jQuery.post/#jqxhr-object, so the rule may be going too far. 

imho disabling one rule in one file is the least worst option (well, i suppose there's the disabling it for specific lines). lint is just a tool that's there to be helpful, not a religion, and even it recognises that you may not want a rule. https://eslint.org/docs/rules/dot-location#when-not-to-use-it
i'd argue that changing the setting globally just to get one file to pass is more disruptive. and i did fix the other errors lint threw up.